Permissioned DEX XLS.md

```diff
@@ -0,0 +1,467 @@
+
+#### 1.3.2. Trading Scenario 1
+
+* Tracy places a permissioned offer on the XRP-USD orderbook. There are no other offers on that orderbook that are a part of the domain.
+* Marko notices that there is a significant price difference between Tracy's offer and the rest of the orderbook. He now submits an open offer to cross and fill Tracy's offer.
+
+#### 1.3.3. Trading Scenario 2
+* Marko has placed open offers on the XRP-EUR orderbook.
+* Tracy places an offer on the XRP-EUR orderbook that crosses one of Marko's offers. Her offer is filled by his.
+
+### 1.4. How Domains Work
+
+A permissioned offer will only be filled by valid domain offers.
+
+An open offer can be filled by any offer on the open DEX, but can _also_ be filled by any permissioned offer, if the open offer is a valid domain offer.
+
+The fact that traders don't need to place a permissioned offer in order to fill offers in that domain enables arbitrage and mixing of liquidity much more easily, as traders do not have to place multiple offers in multiple domains. It also prevents the creation of walled gardens of liquidity.
+
+## 2. On-Ledger Object: `PermissionedDomain`
+
+This object represents a DEX domain.
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerIndex`| ✔️|`string`|`Hash256`|The unique ID of the ledger object.|
+|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`PermissionedDomain`).|
+|`Owner`| ✔️|`string`|`AccountID`|The account that controls the settings of the domain.|
+|`Sequence`|✔️|`number`|`UInt32`|The `Sequence` value of the `OfferCreate` transaction that created this offer. Used in combination with the `Account` to identify this offer.|
+|`AcceptedCredentials`| |`array`|`STArray`|The credentials that are accepted by the domain. Ownership of one of these credentials automatically makes you a member of the domain.|
+|`AcceptedTokens`| |`array`|`STArray`|The tokens that are allowed by the domain.|
+
+#### 2.1.1. `LedgerIndex`
+
+The ID of this object will be a hash that incorporates the `Owner` and `Sequence` fields, combined with a unique space key for `PermissionedDomain` objects, which will be defined during implementation.
+
+#### 2.1.2. `AcceptedCredentials`
+
+This is an array of `Credentials` objects. The maximum length of this array is 10.
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`Issuer`|✔️|`string`|`AccountID`|The issuer of the credential.|
+|`CredentialType`| |`number`|`UInt32`|A value to identify the type of credential from the issuer.|
+
+#### 2.1.3. `AcceptedTokens`
+
+This is an array of `Issue` objects, which represent issued currencies on the XRPL. The maximum length of this array is 10.
```

Comment:
I think `MPT` should be also supported. `MPT` is identified by `MPTIssuanceID`.

Reply:
Ok. I know there's going to be a new `Amount` type that's a superset of `MPTAmount` and `IssuedCurrencyAmount`, is there going to be something similar for `Issue`?

Reply:
Tentatively it's `MPTIssue`.